app/models/admin/import.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class Admin::Import
+  extend ActiveModel::Callbacks
+  include ActiveModel::Model
+  include Paperclip::Glue
+
+  FILE_TYPES = %w(text/plain text/csv application/csv).freeze
+  MODES = %i(merge overwrite).freeze
```

Comment:
You've copied `MODES` from the user import/exports model but are not making use of them anywhere (I'm not advocating for adding support for them, I think they are rather confusing and error-prone).

Reply:
Good point - I was initially planning to support them, then decided not to, and missed removing these